config/health_care_application.yml

```diff
@@ -0,0 +1,30 @@
+endpoints:
```

Comment:
Actually yeah, better do this before merge otherwise all envs are going to use the production endpoint.